<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Drugdictionary</class>
 <widget class="QWidget" name="Drugdictionary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>药品字典</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>增加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="inputButton">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>导入</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="outputButton">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>导出</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteAllButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>全删</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>NO.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>编码</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>简称名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>默认分配仓库</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>规格型号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>生产厂商</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>基本单位</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>转换倍率</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>最小单位</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>加价百分比</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>门诊收据</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>住院收据</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>门诊分类</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>住院分类</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>字典分类</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>门诊单价</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>门诊批发单价</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>住院单价</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>停用标志</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>部门提成百分比</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>员工提成百分比</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>备注信息</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
